tools/Azure.Mcp.Tools.AzureManagedLustre/src/Commands/FileSystem/SubnetSize/SubnetSizeValidateCommand.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine.Parsing;
+using System.Net;
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Tools.AzureManagedLustre.Options;
+using Azure.Mcp.Tools.AzureManagedLustre.Options.FileSystem;
+using Azure.Mcp.Tools.AzureManagedLustre.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.AzureManagedLustre.Commands.FileSystem;
+
+public sealed class SubnetSizeValidateCommand(ILogger<SubnetSizeValidateCommand> logger)
+    : BaseAzureManagedLustreCommand<SubnetSizeValidateOptions>(logger)
+{
+    private const string CommandTitle = "Validate AMLFS subnet against SKU and size";
+
+    public override string Name => "validate";
+
+    public override string Description =>
+        "Validates that the provided subnet can host an Azure Managed Lustre filesystem for the given SKU and size.";
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new()
+    {
+        Destructive = false,
+        Idempotent = true,
+        OpenWorld = false,
+        ReadOnly = true,
+        LocalRequired = false,
+        Secret = false
+    };
+
+    private static readonly string[] AllowedSkus = [
+        "AMLFS-Durable-Premium-40",
+        "AMLFS-Durable-Premium-125",
+        "AMLFS-Durable-Premium-250",
+        "AMLFS-Durable-Premium-500"
+    ];
+
+    private readonly Option<string> _skuOption = AzureManagedLustreOptionDefinitions.SkuOption;
+    private readonly Option<int> _sizeOption = AzureManagedLustreOptionDefinitions.SizeOption;
+    private readonly Option<string> _subnetIdOption = AzureManagedLustreOptionDefinitions.SubnetIdOption;
+    private readonly Option<string> _locationOption = AzureManagedLustreOptionDefinitions.LocationOption;
```

Comment:
nit: New guidance is to stop using this pattern and to just use the Option directly in RegisterOptions and BindOptions